vmdb/app/controllers/ems_common.rb

```diff
@@ -552,6 +556,9 @@ def tree_add_child_nodes(id)
   # Validate the ems record fields
   def valid_record?(ems)
     @edit[:errors] = Array.new
+    if ems.emstype == "scvmm" && ems.security_protocol == "kerberos" && ems.realm.blank?
```

Comment:
Can we just do it outside the PoC work?  This really looks like a validation, so it should be inside the model.  Why are we doing it outside the model?